### DISCUSSION TITLE
installing php-fmp

### DISCUSSION THREAD

**Discussion by Timofej8080** [leokhoa/laragon#569] [Q&A]
How to install php7.4-fpm.sock ? Or which version is installed in Largon?

**Comment by doraemonxxx** [comment]
same question here 🥺

**Comment by Pen-y-Fan** [comment]
Install the NTS version of PHP and Laragon will switch to Apache to use CGI/FastCGI. See [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/)

**Reply by doraemonxxx** [reply]
Problem is laragon doesn't have a php-fpm pools config like on linux. It is hard to understand on how to configure it since most of the articles or answer are based on linux for example *pm.max_child*
